draft-ietf-tls-ticketrequests.md

```diff
@@ -162,6 +162,15 @@ and use tickets beyond common lifetime windows of, e.g., 24 hours. Despite ticke
 hints provided by servers, clients SHOULD dispose of pooled tickets after some reasonable
 amount of time that mimics the ticket rotation period.
 
+In some cases, a server may send NewSessionTicket messages immediately upon sending 
+the server Finished message rather than waiting for the client Finished. If the server
+has not verified the client's ownership of the IP address, e.g., with the TLS 
+Cookie extension (see {{RFC8446}}; Section 4.2.2), an attacker may take advantage of this behavior to create
+an amplification attack proportional to the count value toward a target by performing a key
+exchange over UDP with spoofed packets. Server should limit the number of NewSessionTicket messages
```

Comment:
Overall I like this PR but that last sentence doesn't parse for me. What does it mean?

Reply:
I think the suggestions got messed up -- it should be fine with the proposed edits!

Reply:
I think I'm somewhat understanding it, but it still feels overly complicated. How about we replace the last sentence with `Servers SHOULD limit the number of NewSessionTicket messages they send until they have verified the client's ownership of its IP address.`

Reply:
@mglt can we go with David's suggestion above?